codespell_lib/data/dictionary.txt

```diff
@@ -21252,6 +21260,7 @@ musel->muscle, mussel,
 musels->muscles, mussels,
 musil->muscle, mussel,
 musils->muscles, mussels,
+musl->must
```

Comment:
I suggest you drop this or you move to code. May have significant impact to many.
**musl** is C standard library normally uses in embedded Linux (OpenWRT).
https://musl.libc.org/
https://en.wikipedia.org/wiki/Musl

Reply:
[5b1ecbc](https://github.com/codespell-project/codespell/pull/2430/commits/5b1ecbcfd44c3e632f118ab3a72b92691e2b6dd3)